pkg/storaged/crypto-keyslots.jsx

```diff
@@ -658,7 +658,7 @@ const RemovePassphraseField = (tag, key, dev) => {
                         <Checkbox id="force-remove-passphrase"
                                   isChecked={val !== false}
                                   label={_("Confirm removal with an alternate passphrase")}
-                                  onChange={checked => change(checked ? "" : false)}
+                                  onChange={(_event, checked) => change(checked ? "" : false)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230425-151143-725ccf60-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)